infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -193,8 +193,8 @@ public SequencerServer(ServerContext serverContext) {
     * @param conflictParam The conflict parameter.
     * @return A conflict hash code.
     */
-    public String getConflictHashCode(UUID streamId, int conflictParam) {
-        return streamId.toString() + Integer.toString(conflictParam);
+    public String getConflictHashCode(UUID streamId, byte[] conflictParam) {
+        return streamId.toString() + Utils.bytesToHex(conflictParam);
```

Comment:
Eventually, I think this needs to be removed in general so I didn't really touch it much. Allocating a whole byte array just to do a lookup every time seems pricey.